README.md

```diff
@@ -177,12 +177,20 @@ The client specified MUST implement functions called `getObject` and `putObject`
 
 Sets the `Cache-Control` header on the uploaded files.
 
+This can also be a function. The function will be called for each file, with the
```

Comment:
@martndemus This kind of breaks how deploy plugins config works. It is a feature of ember-cli-deploy that you can specify a config property as a function which will be evaluated at runtime with the current deploy `context` object passed in as an argument.

Therefore, changing how this works for these two config properties seems bad.

Is there any reason why aliasing the plugin is not desirable?